README.md

```diff
@@ -90,53 +335,6 @@ Visit <http://styleguide.pivotal.io> of host the styleguide files with a web ser
 
 then visit <http://localhost:8000>
 
-# Syntax Highlighting
-
-There are two themes, **dark** and **light**, for syntax highlighting. You can choose a theme by linking to one of the following stylesheets:
-
-* `pivotal-ui/prismjs/prism.css` (for the light theme)
-* `pivotal-ui/prismjs/prism-okaidia.css` (for the dark theme).
-
-You can only include one of these themes at a time.
-
-See the latest styleguide for [examples of syntax highlighting in action](http://styleguide.pivotal.io/all.html#code).
-
-## Syntax Highlighting Example
-
-```html
-
-  <pre>
-	<code class='language-ruby'>
-	  class Foo
-	    def bar
-	    end
-	  end
-	</code>
-  </pre>
-
-```
-
-# React components
```

Comment:
I didn't think we needed these legacy React docs anymore. No team consuming our React components are using the monolith, and we don't want any teams to start w/ the Monolith.

Seem okay?